### DISCUSSION TITLE
Docker issue

### DISCUSSION THREAD

**Discussion by alturismo** [FunkeyFlo/ps5-mqtt#383] [General]
### Description of the issue

Hi,

i run HA as Docker (not HA OS), also mqtt

when i follow the instructions running this addon as seperate Docker, either in host mode nor ir eth0 mode the docker wont startup here with this failure

![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/4474d8c7-8269-4a65-b420-3bf1ee21dbb6)

using docker run

![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/1e9a0bd2-21a8-4f5f-926f-41dbf7b73a79)

with existing files

![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/b564e350-c11b-425e-89ba-a67548758e1f)

i assume it tries to reach the HA instance, so in the end, this only works with the supervised HA Installation ?

for an Info, thanks ahead

### Addon version

none

### Platform

OS UNRaid

Docker Installation HA
Core 2024.2.4
Frontend 20240207.1

Docker Installation mqtt, working nice with HA already
![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/95818702-eee1-42ac-8e0c-3e45bc886b79)

### Logs of the issue (if applicable)

text  error  warn  system  array  login  

curl: (6) Could not resolve host: supervisor
[07:25:47] ERROR: Something went wrong contacting the API
curl: (6) Could not resolve host: supervisor
[07:25:47] ERROR: Something went wrong contacting the API
curl: (6) Could not resolve host: supervisor
[07:25:47] ERROR: Something went wrong contacting the API
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: getaddrinfo ENOTFOUND null
Starting PS5-MQTT...
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:109:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'null'
}

** Press ANY KEY to close this window **

**Comment by FunkeyFlo** [comment]
Your custom `run.sh` script isn't being loaded.

**Comment by alturismo** [comment]
> Your custom `run.sh` script isn't being loaded.

may a hint ? you dont have the entrypoint setted in your docker ?

like /config/run.sh ?

**Comment by alturismo** [comment]
when i start it now like this
![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/fd4e6707-2020-4216-ac73-6476cc6fa989)

i end here currently
![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/0e53814a-df2b-47ca-ae78-27479f68da2a)

**Comment by alturismo** [comment]
> Your custom `run.sh` script isn't being loaded.


and may 1 last question, if it wouldnt be loaded ...

![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/8b4b9abf-6aa5-4939-9f4c-6719abb3e9a1)

there shouldnt be the echo as far as my understanding is ;)
![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/b5e58710-883a-4bb1-8906-5cb49f7b5731)

**Reply by alturismo** [reply]
thanks for pointing, and you are right, i "thought" its a either or ... either all by env variables or options file.

container started now ;) thanks for pointing and sorry.

![image](https://github.com/FunkeyFlo/ps5-mqtt/assets/8406490/ae55ee1f-3f27-4ed0-8f63-a3c7136bd4d3)

**Comment by FunkeyFlo** [comment]
You're not providing a path to the options.json config. Refer to the instructions for the specifics.

Also; this isn't a bug or an actual issue with the software but rather a support request. As such I'm closing this issue.

If you require further assistance please open a discussion here on GH, check the HA forums or ask your question on the official Discord.
